include/variables.h

```diff
@@ -202,8 +202,6 @@ extern volatile OSTime gRDPTotalTime;
 
 extern ActiveSound gActiveSounds[7][MAX_CHANNELS_PER_BANK]; // total size = 0xA8
 extern u8 gSoundBankMuted[];
-extern u8 D_801333F0;
-extern u8 gAudioSfxSwapOff;
```

Comment:
Could you also remove these externs in `code_800EC960.c`? There seems to be a third set of unneeded externs for these variables